features/item_creation.feature

```diff
@@ -0,0 +1,18 @@
+#language: fr
+#Louis VAUCHEZ
+#Nicolas HUCHON
+#Jade VENAULT
+#Maxime TRINQUARD
```

Comment:
Authors credits are already in  the commit messages. There is no need to add them to the source files.